plugins/datasetsplit_procedure.h

```diff
@@ -0,0 +1,60 @@
+/** datasetsplit_procedure.h                                                   -*- C++ -*-
+    Mathieu Marquis Bolduc, April 3rd 2017
+    This file is part of MLDB. Copyright 2017 mldb.ai inc. All rights reserved.
+*/
+
+#pragma once
+
+#include "types/value_description_fwd.h"
+#include "server/mldb_server.h"
+#include "mldb/core/procedure.h"
+#include "sql/sql_expression.h"
+
+
+namespace MLDB {
+
+
+/*****************************************************************************/
+/*  DATASET SPLIT PROCEDURE CONFIG                                           */
+/*****************************************************************************/
+
+
+struct SplitProcedureConfig : public ProcedureConfig {
+    static constexpr const char * name = "split.train";
```

Comment:
It feels to me that this is not a train procedure but closer to the melt or even the transform procedures.